src/lua-function-list.json

```diff
@@ -291,12 +300,13 @@
     "setRoomWeight": "setRoomWeight(roomID, weight)",
     "setServerEncoding": "setServerEncoding(encoding)",
     "setStrikeOut": "setStrikeOut([windowName], boolean)",
+    "setTextFormat": "setTextFormat(windowName, r1, g1, b1, r2, g2, b2, bold, underline, italics[, strikeout])",
```

Comment:
Actually, since 3.17.0 (#1840) there will also be optional `overline` and `reverse` arguments (`bool`/`number`/`integer`) on the end there, after strikeout.

Reply:
The wiki needs to be updated for these then - this list is auto-generated!

Reply:
@SlySven I have added both of these to wiki. Please elaborate THERE on what actually happens when you activate overline.